src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/Tds/State/TdsCommandContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.Data.SqlClientX.Tds.State
+{
+    internal class TdsCommandContext
```

Comment:
This came because I was trying to separate context info for command/connection.. This is not used by parser as of now, because I'm only writing it for login flow, but in future this can be extended and used by parser. Will remove it as it has no purpose as of now.